src/utils.ts

```diff
@@ -152,3 +152,24 @@ export const fetch = ({
         body: 'Bridge not available',
     });
 };
+
+const TIMEOUT = 200;
+
+type PermissionsStatus = 'notifications';
+
+export const checkPermissionStatus = (
+    feature: PermissionsStatus,
+    params?: {[key: string]: string},
+): Promise<boolean> =>
+    postMessageToNativeApp(
+        {
+            type: 'OS_PERMISSION_STATUS',
+            payload: {
+                feature,
+                params: params,
+            },
+        },
+        TIMEOUT,
+    )
+        .then(({granted}) => granted)
+        .catch(() => false);
```

Comment:
don't catch here, lib users may want to know the reason of failure, do that in webapp